src/devices/bus/pce/pce_slot.cpp

```diff
@@ -260,8 +264,8 @@ std::pair<std::error_condition, std::string> pce_cart_slot_device::call_load()
 
 		if (m_type == PCE_POPULOUS)
 			m_cart->ram_alloc(0x8000);
-		if (m_type == PCE_CDSYS3J || m_type == PCE_CDSYS3U)
-			m_cart->ram_alloc(0x30000);
+
+		m_cart->install_memory_handlers(*m_address_space.target());
 	}
```

Comment:
You should be able to just do `m_cart->install_memory_handlers(*m_address_space);` – object finders are dereferenceable.